css/main.css

```diff
@@ -0,0 +1,5 @@
+* {
+  background-color: black;
+  color: white;
+  font-size: 72px;
```

Comment:
- [ ] Consider adding the font size and color the element `h1`  instead of the `*` selector